src/WebApi/Program.cs

```diff
@@ -25,16 +26,13 @@
     app.UseHsts();
 }
 
+app.MapOpenApi();
```

Comment:
`Program.cs` is pretty clean at the moment.  Do you think we should wrap the Open API stuff up into an extension method?

Reply:
🤷‍♂️ 1 line vs 2 currently. Happy to if you wish. I don't think it is necessary.

Reply:
OK.  Let's leave for now